crates/typst-library/src/math/matrix.rs

```diff
@@ -80,6 +84,36 @@ pub struct MatElem {
     #[default(Some(Delimiter::Paren))]
     pub delim: Option<Delimiter>,
 
+    /// Draws augmentation lines in a matrix.
+    ///
+    /// - `{none}`: No lines are drawn.
+    /// - A single number: A vertical augmentation line is drawn
+    ///   after the specified column number.
+    /// - A dictionary: With a dictionary, multiple augmentation lines can be drawn
+    ///   both horizontally and vertically. Additionally, the style of the lines can be set.
+    ///   The dictionary can contain the following keys:
+    ///   - `hline`: The offsets at which horizontal lines should be drawn. For example, an
+    ///     offset of `2` would result in a horizontal line being drawn after the second
+    ///     row of the matrix. Accepts either an integer for a single line, or an array
+    ///     of integers for multiple lines.
+    ///   - `vline`: The offsets at which vertical lines should be drawn. For example, an
+    ///     offset of `2` would result in a vertical line being drawn after the second
+    ///     column of the matrix. Accepts either an integer for a single line, or an array
+    ///     of integers for multiple lines.
+    ///   - `stroke`: How to stroke the line. See the [line's documentation]($func/line.stroke)
+    ///     for more details. If set to `{auto}`, takes on a thickness of 0.05em and square line caps.
+    ///
+    /// ```example
+    /// $ mat(1, 0, 1; 0, 1, 2; augment: #2) $
+    /// ```
+    ///
+    /// ```example
+    /// $ mat(0, 0, 0; 1, 1, 1; augment: #(hline: 1, stroke: 2pt + green)) $
+    /// ```
+    #[default(None)]
```

Comment:
default is only needed if the default is different from `Default::default()` (and for booleans because it's clearer)
```suggestion
```